src/components/maintenance/maintenance-media-library-tab.yml

```diff
@@ -2,3 +2,39 @@ maintenanceMediaLibraryTab:
   filterPlaceholder:
     en: Filter by title, tag, license or user
     de: Nach Titel, Tag, Lizenz oder Benutzer filtern
+  bulkDeletePanelHeader:
+    en: Bulk delete
+    de: Bulk delete
+  createdBefore:
+    en: Created before
+    de: Erstellt vor
+  datePlaceholder:
+    en: Date
+    de: Datum
+  and:
+    en: and
+    de: und
+  unused:
+    en: never used
+    de: nie benutzt
+  deprecated:
+    en: no longer used
+    de: nicht mehr benutzt
+  delete:
+    en: "Delete media items"
+    de: "Medien löschen"
+  deleteCount:
+    en: "Delete {count, plural, =1 {# media item} other {# media items}}"
+    de: "{count} Medien löschen"
+  usageInfoMarkdown:
+    en: |
+      **Never used** media is not present in any website content, including document history.
+
+      **No longer used** media is not present in any current website content, but is referenced in documents history.\
+      Archived documents are considered no longer used.
+    de: |
+      **Never used** media is not present in any website content, including document history.
+
+      **No longer used** media is not present in any current website content, but is present in documents history.\
+      Archived documents are considered no longer used.
```

Comment:
**Nie benutzt**e Medien werden von keinem Website-Inhalt (einschließlich Dokumenthistorie) referenziert.\

**Nicht mehr benutzt**e Medien werden von keinem aktuellen Website-Inhalt referenziert, befinden sich jedoch in der Dokumenthistorie.\
Archivierte Dokumente werden hierbei als nicht länger benutzt betrachtet.

Reply:
Changed, thank you!